client/src/components/NavBar/NavBar.tsx

```diff
@@ -13,15 +13,24 @@ import AvatarDisplay from '../../components/AvatarDisplay/AvatarDisplay';
 import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
 import DashboardOutlinedIcon from '@material-ui/icons/DashboardOutlined';
 import logo from '../../Images/logo.png';
+import createBoard from '../../helpers/APICalls/createBoard';
+import { useState } from 'react';
+import OptionsDrawer from '../OptionsDrawer/OptionsDrawer';
 
 interface Props {
   loggedInUser: User;
-  handleDrawerToggle?: () => void;
+  handleDrawerToggle: () => void;
 }
 
-const NavBar = ({ loggedInUser }: Props): JSX.Element => {
+const NavBar = ({ loggedInUser, handleDrawerToggle }: Props): JSX.Element => {
   const classes = useStyles();
 
+  const newBoard = () => {
+    createBoard()
```

Comment:
a mix of .then syntax and using await later on (and also part of `createBoard`) - since this is not doing anything other than console logging I would just stick with either converting it to await or just calling `createBoard()`

Reply:
Would you recommend all syntax either uses `async/await` or `.then/.catch`?

Reply:
I think it's a choice the team can make. If you ask me, my personal opinion is to use async/await. The reason is that I'm not a huge fan of callbacks because you can end up with a "callback hell" (http://callbackhell.com/)